devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy

```diff
@@ -153,6 +153,7 @@ def installed() {
 }
 
 def setColorTemperature(value) {
+    // TODO: Use this method for all zigbee white lights, but also change logic in appengine-zigbee to round
```

Comment:
Noticed this is a copy of the truncating logic I just fixed in the DTHs. Only used in this one place, but if we fix it in appengine it could be used for all of these.

Reply:
Since `zigbee.setColorTemperature` truncates instead of round do you want to fix it to round here and then update all the DTHs to call `zigbee.setColorTemperature()` once it's fixed there?

Reply:
Yeah. Seems like the right thing to do.